### PR TITLE
scribe: note-editor with scroll

### PR DESCRIPTION
**Pull request by samueljd**
Note Editor view with the list of notecallers and their content.
Scroll to the specific notecaller on click in the editor. 
Currently the changes in the note editor doesn't roundtrip.

### CODE REVIEW

File: packages/scribe/utils/readUSFM.js
Comment (irahopkinson) [inline packages/scribe/utils/readUSFM.js:1]: nit: is this empty file still needed?

**Review by irahopkinson — APPROVED** [review]
Good work on this. This PR can be merged as is once you fix the **Test / Build** checks below.

I've added some comments but they can be worked on after this is merged.
I couldn't actual run this fully to start with as I didn't have pyhton3 installed. It would be good to add a `/packages/scribe.README.md` to explain how to get up and running.

**Comment by irahopkinson** [comment]
The **Test / Build** checks above seem to be failing on formatting. You can see this by running `nx format:check` from the repo root. You can likely fix it by running `nx format:write`. I think your formatting rules for the `scribe` package may be in conflict with the monorepo general rules.

File: packages/scribe/api/api.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    input_usj = data["usj"]
+    my_parser = USFMParser(from_usj=input_usj)
+    output = my_parser.usfm
+    return jsonify(output)
+
+
+@app.route("/process_usfm", methods=["OPTIONS"])
+def handle_preflight():
+    response = make_response()
+    response.headers.add("Access-Control-Allow-Origin", "http://localhost:5173/")
```
[the remark this answers is not in the document]
Comment (samueljd) [reply packages/scribe/api/api.py]: Removed the py server. using usfm-grammar for usfm-usj conversion.

File: packages/scribe/src/adaptors/editor-usj-adaptor.test.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import { deepEqual } from "fast-equals";
```
[the remark this answers is not in the document]
Comment (samueljd) [reply packages/scribe/src/adaptors/editor-usj-adaptor.test.ts]: I"ve added it.

File: packages/scribe/utils/readUSFM.js
Comment (samueljd) [reply packages/scribe/utils/readUSFM.js:1]: Thanks. Removed the file.

File: packages/scribe/src/adaptors/editor-usj.adaptor.ts
[the remark this answers is not in the document]
Comment (samueljd) [reply packages/scribe/src/adaptors/editor-usj.adaptor.ts:1]: Yes, they can be. But I still haven't worked on the MarkNode. So Keeping it separate for now.

File: packages/scribe/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "devDependencies": {
+    "@lexical/react": "^0.13.1",
+    "@types/react": "^18.2.56",
+    "@types/react-dom": "^18.2.19",
+    "@vitejs/plugin-react": "^4.2.1",
+    "autoprefixer": "^10.4.18",
+    "concurrently": "^8.2.2",
+    "lexical": "^0.13.1",
+    "platform": "*",
```
[the remark this answers is not in the document]
Comment (samueljd) [reply packages/scribe/package.json]: removed.

File: packages/scribe/package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  "name": "scribe",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "start": "concurrently \"vite\" \"npm run start-python\"",
+    "start-react": "cd client && npm start",
+    "start-python": "cd api && python3 api.py",
+    "dev": "vite",
+    "build": " vite build",
```
[the remark this answers is not in the document]
Comment (samueljd) [reply packages/scribe/package.json]: removed.

**Pull request merged**